Author: Ryan Niebur
Description: par.pl was renamed to par-archive in order to be
 compliant with Debian Policy

--- a/lib/PAR/Packer.pm
+++ b/lib/PAR/Packer.pm
@@ -1237,9 +1237,9 @@
 
     if ($opt->{P}) {
         # write as script
-        $parl = 'par.pl';
+        $parl = 'par-archive';
         unless ( $parl = $self->_can_run($parl, $opt->{P}) ) {
-            $self->_die("par.pl not found");
+            $self->_die("$parl not found");
         }
         $self->{parl} = $parl;
     }
